docs/user_manual/workflows.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: page
+title: "24. Workflows"
+section: 24
+---
+
+
+Workflows are series of pre-defined steps that transition one or more items from a defined starting status to
+a defined end status. They represent a series of steps which would be completed in the lab in order to do some
+amount of work.
+
+{% assign sub = 1 %}
+{% assign figure = 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Workflows Widget" %}
+
+To get to the Workflows widget, click on either the MISO logo or the Home tab in the top left of the page.
+This will take you to the Home page, which contains the Workflows widget. The Workflows (figure {{figure}})
```

Comment:
Could you add a subsection right at the top of the General Navigation section to describe the Home Page and how to get there?

Then remove the "how to get there" part from here and just say the widget is on the home screen